.sync/azure_pipelines/MuDevOpsSync.yml

```diff
@@ -0,0 +1,90 @@
+## @file
+# Azure Pipeline build file for a build using mu_devops.

```

Comment:
I don't know if the fact that the file is synced from `mu_devops` should be its defining characteristic (its filename).

Maybe something like `MuDevOpsWrapper.yml` would be a little more precise when looking at the file on its own in a given repo.

Reply:
thanks, will udpate!

Reply:
The file name is updated.